cmake/CMakeLists.txt

```diff
@@ -253,7 +253,6 @@ option(onnxruntime_FORCE_GENERIC_ALGORITHMS "Disable optimized arch-specific alg
 
 option(onnxruntime_USE_TENSORRT_INTERFACE "Build ONNXRuntime shared lib which is compatible with TensorRT EP interface" OFF)
 option(onnxruntime_USE_NV_INTERFACE "Build ONNXRuntime shared lib which is compatible with NV EP interface" OFF)
-option(onnxruntime_USE_CUDA_INTERFACE "Build ONNXRuntime shared lib which is compatible with Cuda EP interface" OFF)
```

Comment:
Note: I removed CUDA from `--enable_generic_interface`. Doing cuda properly will take a lot more work. I don't think we need CUDA for now anyway.